paperqa/settings.py

```diff
@@ -20,6 +20,7 @@
     model_validator,
 )
 from pydantic_settings import BaseSettings, CliSettingsSource, SettingsConfigDict
+from typing_extensions import deprecated
```

Comment:
We don't have `typing_extensions` here, let's not use this

Just use the built-in `DeprecationWarning` as we do elsewhere in the code base. Imo it's not worth adding another dep for this

Reply:
Ok - followed examples from below